src/components/Asset/Edit/_types.ts

```diff
@@ -1,10 +1,17 @@
+export interface FileInfo {
```

Comment:
can't we use the FileInfo type from oceanJs instead of havin a new type in the market?